docs/sources/configure-client/grafana-agent/ebpf/setup-docker.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "Setup eBPF Profiling on Docker"
+menuTitle: "Setting up on Docker"
+description: "Setting up eBPF Profiling with Grafana Agent on Docker"
+weight: 20
+---
+
+# Setup eBPF Profiling on Docker
+
+To set up eBPF profiling with Grafana Agent on Linux, you need to:
+
+- Verify that your system meets the requirements.
+- Create a Grafana Agent configuration file. For more information, see [Configuration reference][config-reference].
+- Run the Grafana Agent.
+- Finally, verify that profiles are received.
+
+## Prerequisites
+
+Before you begin, you need:
+
+- A Pyroscope server where the agent will send profiling data.
+- Access to Grafana with the [Grafana Pyroscope datasource][pyroscope-ds] provisioned.
+- [Docker Engine](https://docs.docker.com/engine/install/) installed.
+
+> Note: If you don't have a Grafana and/or a Pyroscope server, you can use the [Grafana Cloud][gcloud] free plan to get started.
```

Comment:
We have an admonition shortcode instead of a manual note. 
```suggestion
{{% admonition type="note" %}}
If you don't have a Grafana and/or a Pyroscope server, you can use the [Grafana Cloud][gcloud] free plan to get started.
{{% /admonition %}}
```